tests/integration-tests/conftest_networking.py

```diff
@@ -136,6 +136,19 @@ def get_availability_zones(region, credential):
     return az_list
 
 
+def get_az_setup_for_region(region: str, credential: list):
+    """Return a default AZ ID and its name, the list of all AZ IDs and names."""
+    # TODO Region can be AZ ID, in this case convert it to Region
+    # TODO remove DEFAULT_AVAILABILITY_ZONE
+    az_id_to_az_name_map = get_az_id_to_az_name_map(region, credential)
```

Comment:
Since below you are transforming the results into pairs .. 

for index, (az_id, az_name) in enumerate(zip(az_ids, az_names)):

why not do it here ?

az_pairs = [(id,name) for id,name in az_id_to_az_name_map] ?

Reply:
it's because I was using the two list later on. If you think it's cleaner, I'd rather return the dict

Reply:
There is a slight computational overhead in how the separated lists are created, but this is more a nitpick.

The concern here is "are we assuming that those lists have the same order ?" ...that is the first element of the az_id[1] -> az_name[1] ... and so on ?

If yes, we should keep them together to make the rest of the code more robust.